Minor updates to the Packages docs (#4190)
umbraco/UmbracoDocs: Extending/Packages/UmbPack/index.md

UmbPack is an open source NuGet tool that can deploy packages to Our, and also help you pack your files into a package.

:::note
**Are you using Umbraco 9+?**

> As of Umbraco 9 all packages needs to be installed through NuGet, and it is no longer to install packages from .zip files.

Reviewer: Should this also reference the blogpost (https://umbraco.com/blog/packages-in-umbraco-9-via-nuget/) explaining this change?

```suggestion
As of Umbraco 9, packages can only be installed using NuGet and it is no longer possible to install packages from package zip files.
```
